packages/components/icon/src/icon.tsx

```diff
@@ -133,3 +136,9 @@ export const ChevronIcon: FC<IconProps> = (props) => {
     </Icon>
   )
 }
+
+CheckIcon.displayName = "CheckIcon"
```

Comment:
Please define `__ui__` for each component as well.

Reply:
its done.